From 5f3f17a9c64e33fb787a79dc94d902147da77d71 Mon Sep 17 00:00:00 2001
From: Noel Grandin <noelgrandin@gmail.com>
Date: Sat, 19 Sep 2020 20:38:29 +0200
Subject: fix non-pdfium build

Change-Id: Ic1e30a412927748ba58a21cf2ee922cd1a490aa4
Reviewed-on: https://gerrit.libreoffice.org/c/core/+/103040
Tested-by: Jenkins
Reviewed-by: Noel Grandin <noel.grandin@collabora.co.uk>
---
 vcl/source/filter/ipdf/pdfread.cxx | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

(limited to 'vcl/source/filter/ipdf/pdfread.cxx')

diff --git a/vcl/source/filter/ipdf/pdfread.cxx b/vcl/source/filter/ipdf/pdfread.cxx
index 39c2933be8d5..a9b079a75bc3 100644
--- a/vcl/source/filter/ipdf/pdfread.cxx
+++ b/vcl/source/filter/ipdf/pdfread.cxx
@@ -220,7 +220,7 @@ size_t RenderPDFBitmaps(const void* pBuffer, int nSize, std::vector<Bitmap>& rBi
     (void)rBitmaps;
     (void)nFirstPage;
     (void)nPages;
-    (void)fResolutionDPI;
+    (void)pSizeHint;
     return 0;
 #endif // HAVE_FEATURE_PDFIUM
 }
-- 
cgit v1.2.1

